backend/vkt/src/main/java/fi/oph/vkt/service/ClerkEnrollmentService.java

```diff
@@ -52,4 +58,20 @@ public void changeStatus(final ClerkEnrollmentStatusChangeDTO dto) {
 
     auditService.logById(VktOperation.UPDATE_ENROLLMENT_STATUS, enrollment.getId());
   }
+
+  @Transactional
+  public void move(final ClerkEnrollmentMoveDTO dto) {
+    final Enrollment enrollment = enrollmentRepository.getReferenceById(dto.id());
+    enrollment.assertVersion(dto.version());
+
+    final ExamEvent toExamEvent = examEventRepository.getReferenceById(dto.toExamSessionId());
```

Comment:
toExamEvent -> newExamEvent (jos tuo dto:n nimenmuutos tulee tehtyä)

Reply:
Täällä vois muuten halutessa vielä tarkistaa, että jos uusi tilaisuus on sama kuin vanha, ei edes yritettäisi tehdä mitään. Kuinka tarpeellista niin vähän niin ja näin.

Reply:
Niin, ei varmaan mitään merkitystä. Eipä kantaan mikään sitten muutukaan, audittiin tulee merkintä, mutta niin kai pitäisikin tulla.

Reply:
Joo, eiköhän tää ole näinkin ok. Ja en edes miettinyt tuota audit-lokitusta aiemmin